src/weatherreport/src/weatherreport_check_search.erl

```diff
@@ -0,0 +1,56 @@
+%% -------------------------------------------------------------------
```

Comment:
This check doesn't seem to gracefully degrade when there is no local search node (that's the default for CouchDB 3.x).

Is there a way to mark this as an optional check? Or, alter its severity?

Reply:
I agree that it's annoying for the default configuration to emit an error. I initially thought maybe testing for the presence of clouseau would be a solution, but clouseau code is present by default, it's just not up and talking to lucene.

That said, I think it would be pretty easy to change to a warning.